code/modules/library/library_admin.dm

```diff
@@ -0,0 +1,212 @@
+#define LIBRARY_MENU_MAIN     1
+#define LIBRARY_MENU_CKEY     2
+#define LIBRARY_MENU_REPORTS  3
+
+/client/proc/library_manager()
+	set name = "Manage Library"
+	set category = "Admin"
+	set desc = "Manage Flagged Books and Perform Maintenance on the Library System"
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	if(!SSticker)
+		to_chat(usr, "<span class='warning'>The game hasn't started yet!</span>")
+		return
```

Comment:
```suggestion
```
`SSTicker` always exists. You'd have to change the check to something else for this to work